util/topgen.py

```diff
@@ -6,24 +6,23 @@
 """
 import argparse
 import logging as log
+import random
 import subprocess
 import sys
-import random
-
 from collections import OrderedDict
+from copy import deepcopy
 from io import StringIO
 from pathlib import Path
-from copy import deepcopy
 
 import hjson
 from mako import exceptions
 from mako.template import Template
 
 import tlgen
 from reggen import gen_dv, gen_rtl, validate
-from topgen import (amend_clocks, get_hjsonobj_xbars, merge_top, search_ips,
-                    validate_top)
+from topgen import amend_clocks, get_hjsonobj_xbars
 from topgen import intermodule as im
+from topgen import merge_top, search_ips, validate_top
```

Comment:
Is there a reason to use multiple import lines like this, rather than the parenthesized form above? (Is this just some weird linting rule that doesn't play well with the `from X import Y as Z` syntax?)